### PR TITLE
New release: v9.0.0

### PR DESCRIPTION
**Pull request by max-rocket-internet**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,25 +7,29 @@ project adheres to [Semantic Versioning](http://semver.org/).
...
-## [[v8.?.?](https://github.com/terraform-aws-modules/terraform-aws-eks/compare/v8.2.0...HEAD)] - 2020-xx-xx]
-- Revert #631. Add back manage security group flags. (by @ryanooi)
+## [[v9.?.?](https://github.com/terraform-aws-modules/terraform-aws-eks/compare/v9.0.0...HEAD)] - 2020-xx-xx]
+
+- Write your awesome change here (by @you)
+
+# History
 
-- changed timeout for creating EKS (by @confiq)
+## [[v9.0.0](https://github.com/terraform-aws-modules/terraform-aws-eks/compare/v8.2.0...v9.0.0)] - 2020-xx-xx]
```
Comment (barryib) [inline CHANGELOG.md]: Add release date.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,25 +7,29 @@ project adheres to [Semantic Versioning](http://semver.org/).
...
+
+- Revert #631. Add back manage security group flags. (by @ryanooi)
+- Changed timeout for creating EKS (by @confiq)
 - Added instructions for how to add Windows nodes (by @ivanguravel)
 - [CI] Switch `Validate` github action to use env vars (by @max-rocket-internet)
 - [CI] Bump pre-commit-terraform version (by @barryib)
 - Added example `examples/irsa` for IAM Roles for Service Accounts (by @max-rocket-internet)
 - **Breaking:** Removal of autoscaling IAM policy and tags (by @max-rocket-internet)
```
Comment (barryib) [inline CHANGELOG.md]: Move breaking changes to the top. In that way, they are more visible.

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,25 +7,29 @@ project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Next release
 
-## [[v8.?.?](https://github.com/terraform-aws-modules/terraform-aws-eks/compare/v8.2.0...HEAD)] - 2020-xx-xx]
-- Revert #631. Add back manage security group flags. (by @ryanooi)
+## [[v9.?.?](https://github.com/terraform-aws-modules/terraform-aws-eks/compare/v9.0.0...HEAD)] - 2020-xx-xx]
+
+- Write your awesome change here (by @you)
+
+# History
```
Comment (barryib) [inline CHANGELOG.md]: Can you please remove unnecessary "hisotory" sections from the changelog. https://github.com/terraform-aws-modules/terraform-aws-eks/pull/752/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eL36  and https://github.com/terraform-aws-modules/terraform-aws-eks/pull/752/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eL85

File: CHANGELOG.md (hunk 1 above)
Comment (max-rocket-internet) [reply CHANGELOG.md]: Ah yes

**Pull request merged**